scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -1202,3 +1202,46 @@ task PopulateFilterSetInfo {
     File monitoring_log = "monitoring.log"
   }
 }
+
+task GetHailScripts {
+  input {
+    File? hail_generate_sites_only_script_path
+    String variants_docker
+  }
+  meta {
+    # OK to cache this as the scripts are drawn from the stringified Docker image and as long as that stays the same
+    # the script content should also stay the same.
+  }
+  String hail_generate_sites_only_script_name = if (defined(hail_generate_sites_only_script_path)) then basename(select_first([hail_generate_sites_only_script_path])) else "dummy.txt"
```

Comment:
Is this leftover debug code?

Reply:
No. In the absence of optional outputs, I wanted the script to return a valid (but non-existent) file. 
Really not happy with this solution, would welcome a suggestion - even if it's to name the file something else (like 'this_should_never_happen.py'?